crates/autopilot/src/domain/fee/mod.rs

```diff
@@ -56,7 +56,7 @@ impl ProtocolFee {
                     buy: quote.buy_amount,
                     fee: quote.fee,
                 };
-                tracing::debug!(?order.metadata.uid, ?self.policy, ?order_, ?quote, "checking if order is outside market price");
+                tracing::trace!(?order.metadata.uid, ?self.policy, ?order_, ?quote, "checking if order is outside market price");
```

Comment:
> data should be available in the DB to later reproduce the math

Didn't know if this is used for debugging at the moment.
The `policy` is a runtime argument but we could recover that from the initialization logs which prints all the arguments so I would be fine with dropping it entirely.
Since this is not super urgent I'd wait for @sunce86 to comment on this since he is probably the one who uses those logs the most (if at all).

Reply:
This log is the same for an order in all the auctions it appears.

I would prefer to move this log [here](https://github.com/cowprotocol/services/blob/main/crates/shared/src/order_validation.rs#L668-L690)  and do it for every order, as this check will be the basis for determining the class of an order quite soon (so logging it once per order creation seems ok to avoid spending time to query the db and reconstruct the calculation)